File: code/library/src/main/java/org/fbme/lib/iec61499/fbnetwork/EndpointCoordinate.java
package org.fbme.lib.iec61499.fbnetwork;

import org.fbme.lib.common.CompositeReference;
import org.fbme.lib.common.Element;

public interface EndpointCoordinate extends Element {
    int getX();

    void setX(int x);

    int getY();

    void setY(int y);

    CompositeReference<PortPath<?>> getPortReference();
}

File: code/library/src/main/kotlin/org/fbme/lib/iec61499/fbnetwork/EndpointCoordinate.kt
package org.fbme.lib.iec61499.fbnetwork

import org.fbme.lib.common.CompositeReference
import org.fbme.lib.common.Element

interface EndpointCoordinate : Element {
    var x: Int
    var y: Int
    val portReference: CompositeReference<PortPath<*>>
}
